.docs/content/1.concepts/7.adaptors.md

```diff
@@ -9,6 +9,8 @@ It keeps track of the state of the computations and data managed by ArmoniK.
 
 An interface lies between each service and the other components of Core. The adaptors provide implementations of the interfaces. The interface abstracts away all the complexities and the details of the underlying service (queue, database or object storage), and provides the features of the service that Core components may need. A NuGet package [ArmoniK.Core.Base](https://www.nuget.org/packages/ArmoniK.Core.Base/) is generated, it contains the interfaces and classes needed for the implementation of a custom adaptor.
 
```

Comment:
The interface abstracts away all the complexities and the details of the queue, and provides the features of the queue that Core components may need. It is defined in the package [ArmoniK.Core.Base](https://www.nuget.org/packages/ArmoniK.Core.Base/0.14.0-SNAPSHOT.194.5b51131f). This NuGet package contains the interfaces for the queue adaptor as well as other interfaces.